javascript/javascript1/week4/voice_assistant.js

```diff
@@ -0,0 +1,94 @@
+const assistant = {};
+const addToDosList = (assistant.toDos = []);
+
+function getReply(command) {
+
+const getUserName = command.split(" ").pop();
```

Comment:
Consider initializing userName inside the assistant, as a way of knowing the properties it will contain during the whole execution of the program.

Question: what would happen if I "pop" the last word from all of the commands? Then userName would include unwanted words that are not names.

The userName property should only be defined when the command "Hello my name is" is executed, so to avoid bugs, this line could be modified as: userName = command.split(" ").pop( ); and moved to line 13.